### DISCUSSION TITLE
Android 12 - SD maid Pro or SD Maid 2/SE

### DISCUSSION THREAD

**Discussion by Oloooooooo** [d4rken-org/sdmaid-se#238] [Q&A] [answered]
Android 12 - SD maid Pro (bought in 2023 march) or SD Maid 2/SE, which version of SD Maid is better for Android 12?

**Comment by d4rken** [comment, marked as answer]
I would recommend to use SD Maid 2/SE on any Android 8.0+ device.
Scans in SD Maid 2/SE already have better results than SD Maid 1 in most cases. The only downside I currently see is SD Maid 1 being a bit faster, work in progress 🙂 .

An edge-case would be if you want a to use a feature (e.g. Databases, Duplicates), that SD Maid 2/SE does not yet support.

**Reply by Oloooooooo** [reply]
Thanks for answer :)
